### ISSUE TITLE
Plz help me 

### ISSUE THREAD

**Issue by snippy62** [gurnec/btcrecover#258]
So hear it is i sent a little bit of coin to a core wallet back 05.15.17 (0.024BTC) there is a password on the wallet i cant remember putting it there but i must have so in the wallet there is only 3 receaving addresses in the wallet i dont know if this is the best option but ive been trying for a long time i would like so help making the corect file its not complacated but i just cant make it work  hope some one can help me i have changed the words from the tokens file  put youll get the idea so it will contain one or two words out of the 6 words never the same word twice i could have up two 3 nummbers after the second word and up two 3 specal char after the first word somthing like @#$%&*! also in cluding nummber 0-9 but no more than 3 total char at a time after each word 

Cracker cracker
Alberta alberta
Birthday birthday
bnsggg  Bnsggg
Birthdaycake birthdaycake
%0,3d$
%0,3y

C:\python27\python btcrecover.py --wallet wallet.dat --tokenlist tokens.txt --enable-gpu --no-dupchecks --autosave savefile

**Comment by crypto-recovery** [comment]
Apologies if I have misunderstood.  It sounds like you want to create password variations that meet the following criteria:

- Exactly 4 tokens
- Exactly two of the "word" tokens (ex: "Cracker" and "bnsggg")
- 0-3 special characters in second position (after the first "word" token)
- 0-3 digits at the end (position 4, after the second "word" token)

For example:
- CrackerAlberta
- Cracker@Alberta
- Cracker@Alberta0
- Cracker@#Alberta0
- Cracker@#Alberta01
- Cracker@#$Alberta01
- Cracker@#$Alberta012
... and many, many more

Is that right?  But you would never see:
- Cracker@#$012
- Cracker@#$cracker012
- Cracker012cracker@#$

I'd try this in your tokens file in combination with --min-tokens 4 --max-tokens 4:

Cracker cracker
Alberta alberta
Birthday birthday
bnsggg Bnsggg
Birthdaycake birthdaycake
%0,3d$
^2^%0,3y

Thanks,
Chris

**Comment by pedrohrcunha** [comment]
@crypto-recovery [off-topic]: Do you happen to work with wallet recovery? I'm looking for an alternative service on mine, I tried Dave without success.

**Comment by crypto-recovery** [comment]
Yes, you can reach us at CryptoAssetRecovery.com/contact/ or cryptoassetrecovery@gmail.com.

Thanks,
Chris

**Comment by jonathancross** [comment]
Try https://walletrecoveryservices.com -- I don't know then personally, but they have a [good reputation](https://www.reddit.com/r/Bitcoin/search/?q=walletrecoveryservices.com&restrict_sr=1).

**Comment by nori19870522** [comment]
I am Japanese and do password analysis.
https://keychainx.io/ 

This is Robert's article.
https://news.bitstarz.com/bitstarz-news-interview-robert-rhodin-keychainx-ceo  

I may be able to find the password you lost. Do you need help with your login?

**Comment by btcexchanging** [comment]
if you haven't recovered your wallet, i can help you recover your wallet.
I have recovered most of the common wallets and recovery phrases for many users
you can reach me here at my telegram account:  @walletrecover

**Comment by viladmir2022** [comment]
hi, if you still have wallet problem. I can recover your lost wallet.
I have powerful servers and excellent software that can find any password.
and my fee is only 15-25% of the recovered fund
here is my telegram id: @viladmir2022  and Gmail: walletsrecoveryservice@gmail.com
